tests/test_cli.py

```diff
@@ -358,6 +359,103 @@ def my_client(rtype, *args):
         assert 'ExtraParam: extra value\\n' in result.output
 
 
+def test_print_taupage_config_without_ref(monkeypatch):
+    def my_resource(rtype, *args):
+        if rtype == 'ec2':
+            ec2 = MagicMock()
+            ec2.security_groups.filter.return_value = [MagicMock(name='app-master-mind', id='sg-007')]
+            return ec2
+        return MagicMock()
+
+    monkeypatch.setattr('boto3.client', MagicMock())
+    monkeypatch.setattr('boto3.resource', my_resource)
+    data = {'SenzaInfo': {'StackName': 'test',
+                          'Parameters': [{'ApplicationId': {'Description': 'Application ID from kio'}}]},
+            'SenzaComponents': [{'Configuration': {'ServerSubnets': {'myregion': ['subnet-123']},
+                                                   'Type': 'Senza::Configuration'}},
+                                {'AppServer': {'Image': 'AppImage',
+                                               'InstanceType': 't2.micro',
+                                               'SecurityGroups': ['app-{{Arguments.ApplicationId}}'],
+                                               'IamRoles': ['app-{{Arguments.ApplicationId}}'],
+                                               'TaupageConfig': {'runtime': 'Docker',
+                                                                 'source': 'foo/bar',
+                                                                 'mint_bucket': 'zalando-mint-bucket',
+                                                                 'environment': {'ENV1': 'v1',
+                                                                                 'ENV2': 'v2'}},
+                                               'Type': 'Senza::TaupageAutoScalingGroup'}}]
+            }
+
+    runner = CliRunner()
+
+    with runner.isolated_filesystem():
+        with open('myapp.yaml', 'w') as fd:
+            yaml.dump(data, fd, default_flow_style=False)
+
+        result = runner.invoke(cli, ['print', 'myapp.yaml', '--region=myregion', '123', 'master-mind'],
+                               catch_exceptions=False)
+
+        output = io.StringIO(result.output)
+        output.readline()
+        awsjson = json.load(output)
+
```

Comment:
Hi @tuxlife, you merged before my last commit to address this: 
https://github.com/chris-zen/senza/commit/49ab814a46f4c5666a897ac4107aaec454b13edc